app/models/post.rb

```diff
@@ -1,3 +1,17 @@
 class Post < ApplicationRecord
-  belongs_to :user, foreign_key: 'author_id'
+  belongs_to :author, class_name: 'User', foreign_key: 'author_id'
```

Comment:
- [ ] The `foreign_key: 'author_id'` should be added in the user model as required in the comment above, please remove it to avoid issues.